crates/ruff/src/rules/flake8_pyi/rules/stub_body_multiple_statements.rs

```diff
@@ -0,0 +1,50 @@
+use rustpython_parser::ast::{self, Constant, Expr, Stmt};
+
+use crate::checkers::ast::Checker;
+use ruff_diagnostics::{Diagnostic, Violation};
+use ruff_macros::{derive_message_formats, violation};
+use ruff_python_ast::helpers;
+
+#[violation]
+pub struct StubBodyMultipleStatements;
+
+impl Violation for StubBodyMultipleStatements {
+    #[derive_message_formats]
+    fn message(&self) -> String {
+        format!("Function body must contain exactly 1 statement")
+    }
+}
+
+/// PYI010
+pub(crate) fn stub_body_multiple_statements(checker: &mut Checker, stmt: &Stmt, body: &[Stmt]) {
+    match body.len() {
+        1 => (),
+        2 => {
+            // If 2 statements and one is a docstring. Skip as this is covered by PYI021
+            if is_docstring(&body[0]) {
```

Comment:
We have an existing `is_docstring_stmt` helper in the `ruff_python_ast` crate -- can we use that here?